apis/Google.Monitoring.V3/Google.Monitoring.V3.Snippets/GroupServiceClientSnippets.cs

```diff
@@ -43,7 +43,7 @@ public void ListGroups()
             {
                 Console.WriteLine($"{group.Name}: {group.DisplayName}");
             }
-            // End snippet
+            // FIXME:End snippet
```

Comment:
Yes - we don't really understand comments. Previously the start snippet bit tripped us up... now the end snippet does. The FIXME stops us from treating it as a snippet at all.